AI-and-Analytics/Getting-Started-Samples/IntelAIKitContainer_GettingStarted/README.md

```diff
@@ -0,0 +1,180 @@
+﻿# Intel&reg; oneAPI AI Analytics Toolkit Container Sample
+
+Containers allow you to set up and configure environments for
+building, running, and profiling oneAPI AI applications and distribute
+them using images. You can also use Kubernetes* to automate the
+deployment and management of containers in the cloud.
+
+This get started sample shows the easiest way to start using any of
+the [Intel® oneAPI AI Analytics
+Toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit.html)
+components without the hassle of installing the toolkit, configuring
+networking and file sharing.
+
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04 
+| Hardware                          | Intel® Xeon® Scalable processor family or newer
+| Software                          | Intel® AI Analytics Toolkit
+| What you will learn               | How to start using the Intel® oneapi-aikit container
+| Time to complete                  | 10 minutes
+
+## Purpose
+
+This sample provides a Bash script to help users configure their Intel
+oneAPI AI Analytics Toolkit container environment. Developers can
+quickly build and train deep learning models using this Docker*
+environment.
+
+For more information on the one API AIKit container, see [oneAPI AIKit
+Container Repository](https://hub.docker.com/r/intel/oneapi-aikit).
+
+
+## Key Implementation Details
+
+The Bash script provided in this sample performs the following
+configuration steps:
+
+- Mounts the `/home` folder from host machine into the Docker
+  container. You can share files between the host machine and the
+  Docker container via the `/home` folder.
+
+- Applies proxy settings from the host machine into the Docker
+  container.
+   
+- Uses the same IP addresses between the host machine and the Docker
+  container.
+
+- Forwards ports 8888, 6006, 6543, and 12345 from the host machine to
+  the Docker container for some popular network services, such as
+  Jupyter notebook and TensorBoard.
+        
+
+## Run the Sample
+
+This sample uses a configuration script to automatically configure the
+environment. This provides fast and less error prone setup. For
+complete instructions for using the oneAPI AI Analytics containers see
+the [Getting Started Guide]
+(https://www.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux/top/using-containers.html.)
```

Comment:
Please delete a space between ] and (.

[Getting Started Guide](https://www.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux/top/using-containers.html.)

Reply:
fixed it. thanks for identifying the problem.